### PR TITLE
Changes to work with Oga

### PR DESCRIPTION
**Pull request by LightGuard**
I've updated to the branch with Oga instead of REXML. Also I've disabled
livereload as that branch doesn't have it. Changed embed to not have a closing
tag. blog/2007-10-25-eclipse-the-memory-hog---resolved.html.erb had an issue 
with some invalid html syntax, I fixed that up as well.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -46,5 +46,6 @@ gem 'asciidoctor', '>= 1.5.2'                           # If using AsciiDoc synt
 # gem 'kramdown', '>= 0.14.2'                             # If using Markdown
 # gem 'uglifier', '>= 1.3.0'                              # If using the minify transformer
 # gem 'htmlcompressor', '>= 0.0.3'                        # If using the minify transformer
+gem 'bootstrap-sass'
```
Comment (maxandersen) [inline Gemfile]: any specific reason you wanted to add this ?

File: blog/2007-10-25-eclipse-the-memory-hog---resolved.html.erb [hunk 2]
```diff
@@ -14,7 +14,7 @@ Why were so many seeing memory issues on a release that actually should use less
 <br><br>
 As it turns out, on top of the issues with eclipse.ini from my previous entry, Eclipse 3.3.1 introduced <a href="https://bugs.eclipse.org/bugs/show_bug.cgi?id=203325">a major bug</a> in parsing the settings in eclipse.ini that resulted in PermGen memory settings not being set correctly on Sun Java VM's! 
 <br><br><b>Result:</b> Eclipse 3.3.1 runs "fine" at first, but as soon as you start having larger projects and/or using additonal plugins you are bound to get OutOfMemoryExceptions! 
-<br><br><b>Thankfully</b> eclipse.org is now in the process of doing a respin of Eclipse 3.3.1 and the first build is now <a href="&lt;a%20href=" http:>available for testing</a>.
+<br><br><b>Thankfully</b> eclipse.org is now in the process of doing a respin of Eclipse 3.3.1 and the first build is now <a href="http:">available for testing</a>.
```
Comment (maxandersen) [inline blog/2007-10-25-eclipse-the-memory-hog---resolved.html.erb]: good catch - didn't see that one ;)

**Comment by maxandersen** [comment]
merged this in. thanks!

File: Gemfile (hunk 1 above)
Comment (LightGuard) [inline Gemfile]: Aren't you using bootstrap? In the next release, Awestruct won't depend on Bootstrap, or foundation, or 960, so you'll need it in your Gemfile.

**Pull request closed without merging**